src/react-super-select.js

```diff
@@ -247,7 +247,9 @@ class ReactSuperSelect extends React.Component {
 
   // Update focused element after re-render
   componentDidUpdate(prevProps, prevState) {
-    this._focusCurrentFocusedId();
+	if (this.props.disabled) {
```

Comment:
are you checking the correct prop here?

Reply:
No, you right! Should be `focusToSelectedValue` of course. It's because of copy-paste to keep your style, was sure that I replaced that.